src/frontend/app/shared/components/list/list-types/app-route/cf-app-routes-list-config.service.ts

```diff
@@ -71,47 +79,68 @@ export class CfAppRoutesListConfigService implements IListConfig<EntityInfo> {
     label: 'Unmap',
     description: 'Unmap route',
     visible: (row: EntityInfo) => true,
-    enabled: (row: EntityInfo) => true,
+    enabled: (row: EntityInfo) => true
   };
 
   private listActionAdd: IGlobalListAction<EntityInfo> = {
     action: () => {
-      this.store.dispatch(new RouterNav({ path: ['applications', this.appService.cfGuid, this.appService.appGuid, 'add-route'] }));
+      this.store.dispatch(
```

Comment:
At some point you might struggle with code-climate line length warnings with these kind of expansions. If it's a plugin it might be worth disabling that rule